test/facade_converter_test.ts

```diff
@@ -36,8 +36,23 @@ function getSources(str: string): {[k: string]: string} {
     'angular2/src/core/di/forward_ref.d.ts': `
         export declare function forwardRef<T>(x: T): T;`,
     'angular2/typings/es6-promise/es6-promise.d.ts': `
-        declare class Promise<R> {}
-        declare module Promise {}`,
+      interface Thenable<R> {
+        then<U>(onFulfilled?: (value: R) => U | Thenable<U>,  onRejected?: (error: any) => U | Thenable<U>): Thenable<U>;
```

Comment:
Re all the comments below, a perfectly fine answer is "I'm just copying whatever is in lib.d.ts". But then please add a comment.